### PR TITLE
Some improvements to the UI crate

### PR DESCRIPTION
**Pull request by torkleyy**
* Move shaders to subfolder
* Format some code
* Rename `FontFormat`
* Fix nits in docs

### CODE REVIEW

File: amethyst_ui/src/text.rs [hunk 1]
```diff
@@ -25,6 +26,11 @@ impl UiText {
     /// Initializes a new UiText
     ///
     /// # Parameters
+    ///
+    /// * `font`: A handle to a `Font` asset
+    /// * `text`: the glyphs to render
+    /// * `color`: normalized RGBA color (meaning R + G + B = `1.0`)
```
Comment (torkleyy) [inline amethyst_ui/src/text.rs]: Not sure if this is true, but I think alpha isn't part of this sum, is it?

Comment (Xaeroxe) [reply amethyst_ui/src/text.rs]: No that's not what I meant, sorry the word normalized here is confusing.  I meant each channel has a max of 1

File: amethyst_ui/src/format.rs [hunk 2]
```diff
@@ -27,10 +27,10 @@ impl Into<Result<FontAsset, Error>> for FontData {
 /// exist in TrueType this will fail.  This will only load the first font contained in a file.
 /// If this is a problem for you please file an issue with Amethyst on GitHub.
 #[derive(Clone)]
-pub struct FontFormat;
+pub struct TtfFormat;
```
Comment (Xaeroxe) [inline amethyst_ui/src/format.rs]: This is a little misleading, as it can load Otf files too.

Comment (torkleyy) [reply amethyst_ui/src/format.rs]: Then maybe we could just do two formats?

Comment (Xaeroxe) [reply amethyst_ui/src/format.rs]: That are both functionally identical?  Perhaps we could use a type alias?

**Comment by Xaeroxe** [comment]
I fixed the stuff I mention here and merged it.  Thanks!

**Pull request merged**